clients/da-vinci-client/src/main/java/com/linkedin/davinci/replication/merge/MergeConflictResolver.java

```diff
@@ -766,9 +766,11 @@ private boolean ignoreNewUpdate(
       case PER_FIELD_TIMESTAMP:
         GenericRecord timestampRecord = (GenericRecord) oldTimestampObject;
```

Comment:
unrelated nit: can we rename this as `oldTimestampRecord` ?

Reply:
Sure, I can update.